<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>symbolEffectDialog</class>
 <widget class="QDialog" name="symbolEffectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>431</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>431</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>460</width>
    <height>440</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>391</y>
     <width>231</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>250</width>
     <height>411</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(64, 64, 64);
border: 2px solid rgb(16, 16, 16);
border-radius: 4px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>0</number>
     </property>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_left">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>274</x>
     <y>302</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Left</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_up">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>283</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_down">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>321</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Down</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_right">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>376</x>
     <y>302</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Right</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>288</x>
     <y>176</y>
     <width>59</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Color:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_color">
   <property name="geometry">
    <rect>
     <x>334</x>
     <y>176</y>
     <width>87</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #ffffff</string>
   </property>
   <property name="text">
    <string>#ffffff</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Select a symbol</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Question Mark</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Exclamation Mark</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dollar Sign</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Percent Sign</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Left Parenthesis</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Right Parenthesis</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Left Square Bracket</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Right Square Bracket</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Plus Sign</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Minus Sign</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Equals Sign</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Up Caret</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Left Caret</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Right Caret</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Left Hook Bracket</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Right Hook Bracket</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Forward Slash</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Backward Slash</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Semicolon</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Colon</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Asterisk</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bar</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tetris Piece I</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tetris Piece O</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tetris Piece T</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tetris Piece S</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tetris Piece Z</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tetris Piece J</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tetris Piece L</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>symbolEffectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>symbolEffectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
